docs/cli/storage.rst

```diff
@@ -93,7 +93,8 @@ the "mydata1" virtual folder.
 
 In your code, you may access the virtual folder via ``/home/work/mydata1``
 (where the default current working directory is ``/home/work``) just like
-a normal directory.
+a normal directory. If you want to mount vfolders in other path, add '/'
```

Comment:
```suggestion
a normal directory.  If you want to mount vfolders in other path, add '/'
```

In many plain-text based writing, sentences are often distinguished using two spaces.